test/qweb3.js

```diff
@@ -289,13 +296,19 @@ describe('Qweb3', () => {
   /** ******** RAW TRANSACTIONS ********* */
   describe('getHexAddress()', () => {
     it('returns the hex address', async () => {
-      assert.equal(await qweb3.getHexAddress(QTUM_ADDRESS), '17e7888aa7412a735f336d2f6d784caefabb6fa3');
+      const hexDecodedaAddress = bs58.decode(QTUM_ADDRESS).toString('hex')
+      var hexadecimalAddress = await qweb3.getHexAddress(QTUM_ADDRESS)
+      assert.isString(hexadecimalAddress);
+      assert.lengthOf(hexadecimalAddress, 40, `${hexadecimalAddress} has length of 40`);
```

Comment:
remove the third argument (comment). would like to keep the tests clean of any unnecessary code. if it errors, then it will say the length is wrong anyways.

Reply:
Ok, and I'd even say that hardcode the error message can cause future diagnostic problems if not properly maintained. I'll change it!